test/e2e/framework/log/logger.go

```diff
@@ -47,6 +47,17 @@ func Failf(format string, args ...interface{}) {
 // (for example, for call chain f -> g -> FailfWithOffset(1, ...) error would be logged for "f").
 func FailfWithOffset(offset int, format string, args ...interface{}) {
 	msg := fmt.Sprintf(format, args...)
-	log("INFO", msg)
+	log("FAIL", msg)
 	ginkgowrapper.Fail(nowStamp()+": "+msg, 1+offset)
 }
+
+// Fail is a replacement for ginkgo.Fail which logs the problem as it occurs
+// and then calls ginkgowrapper.Fail.
```

Comment:
as a side note, it doesn't seem that we have that many direct calls to ginkgo.Fail:
https://github.com/kubernetes/kubernetes/search?q=%22ginkgo.Fail%22&unscoped_q=%22ginkgo.Fail%22